engines/sci/graphics/frameout.cpp

```diff
@@ -1334,14 +1366,15 @@ void GfxFrameout::kernelSetNowSeen(const reg_t screenItemObject) const {
 
 	ScreenItem *screenItem = plane->_screenItemList.findByObject(screenItemObject);
 	if (screenItem == nullptr) {
-		error("kSetNowSeen: Screen item %04x:%04x not found in plane %04x:%04x", PRINT_REG(screenItemObject), PRINT_REG(planeObject));
+		return false;
```

Comment:
It would make sense to keep a warning for this behavior, at least during initial game playthroughs, with a TODO to remove it. That way, we can at least check that this won't cause unexpected behavior and errors in other parts of the engine.